README.md

```diff
@@ -54,6 +46,14 @@ Requirements:
 NOTE: Make sure you run this from a VS Command Prompt or have run VCVARSALL.BAT in your current shell so
 CMake can find Visual Studio.
 
+## Mac-Only TLS Behavior
```

Comment:
I would put this beneath the Documentation and Installation sections, or as a `####` minor section within the OSX build steps/requirements